elements/k-new-post.html

```diff
@@ -24,68 +51,106 @@
-      </div>
+        <template is="dom-if" if="{{wide}}">
+          <div class="container flex-horizontal">
+            <div class="flex8child">
+              <k-title title="{{titleValue}}"></k-title>
+              <k-editor content="{{contentValue}}"></k-editor>
+            </div>
+            <div class="flex4child">
+              <k-category category="{{categoryValue}}" categories="[[categoryList]]"></k-category>
+              <k-short-content short-content="{{shortContentValue}}"></k-short-content>
+              <k-image-upload
+                class="image-upload"
+                image="{{imageValue}}"
+                image-upload-url="[[headerUploadUrl]]"
+                method="[[uploadMethod]]"
+                thumbnail="{{thumbnailImage}}"
+                uploading="{{uploadingValue}}"
+              ></k-image-upload>
+
+              <iron-image
+                src$="[[thumbnailImage]]"
+                preload
+                fade
+              ></iron-image>
+              <br>
+              <k-publishing
+                title="[[titleValue]]"
+                short-content="[[shortContentValue]]"
+                content="[[contentValue]]"
+                category="[[categoryValue]]"
+                post-url="[[postUrl]]"
+                image="[[imageValue]]"
+                disabled="{{uploadingValue}}"
+              ></k-publishing>
+            </div>
+          </div>
+        </template>
+
+        <template is="dom-if" if="{{!wide}}">
+          <k-title title="{{titleValue}}"></k-title>
+          <k-short-content short-content="{{shortContentValue}}"></k-short-content>
+          <k-category category="{{categoryValue}}" categories="[[categoryList]]"></k-category>
+          <k-image-upload
+            class="image-upload"
+            image="{{imageValue}}"
+            image-upload-url="[[headerUploadUrl]]"
+            method="[[uploadMethod]]"
+            thumbnail="{{thumbnailImage}}"
+            uploading="{{uploadingValue}}"
+          ></k-image-upload>
 
+          <iron-image
+            src$="[[thumbnailImage]]"
+            preload
+            fade
+          ></iron-image>
+          <k-editor content="{{contentValue}}"></k-editor>
+          <br>
+          <k-publishing
+            title="[[titleValue]]"
+            short-content="[[shortContentValue]]"
+            content="[[contentValue]]"
+            category="[[categoryValue]]"
+            post-url="[[postUrl]]"
+            image="[[imageValue]]"
+            disabled="[[uploadingValue]]"
+          ></k-publishing>
+        </template>
+      </div>
       <div class="preview">
-        <div>
-          <b>Title:</b>
-          <span>[[titleValue]]</span>
+        <div style="text-transform: capitalize;">
+          <h2><b>[[titleValue]]</b></h2>
         </div>
 
         <div>
-          <b>Category:</b>
-          <span>[[categoryValue]]</span>
+          <span><b><em>[[shortContentValue]]</em></b></span>
```

Comment:
shouldn't compile two html tags for shortContentValue, instead define the CSS rules instead.